app/assets/stylesheets/log_in.css.scss

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap');
+
+$primaryColor: #AEFEFF;
+$very_pale_cyan: #4FBDBA;
+$very_Dark_Cyan: #072227;
+$normal_Cyan: #35858B;
```

Comment:
I see you are duplicating code [here](https://github.com/BC-MAY-21-ROR/kata-07-reloj-checador-k7-f22-t4/blob/master/app/assets/stylesheets/check.css.scss#L3)
you are using sass, you could create a `variables` file, loading it first and use it in all your sass files..

take a look of the [7-1 pattern](https://sass-guidelin.es/#the-7-1-pattern)

Reply:
Yes, I checked your recommendations and made the changes to avoid duplicate code.